packages/salesforcedx-vscode-core/src/commands/forceRenameLightningComponent.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { LibraryCommandletExecutor } from '@salesforce/salesforcedx-utils-vscode/out/src';
+import { notificationService } from '@salesforce/salesforcedx-utils-vscode/out/src/commands';
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/src/types';
+import * as fs from 'fs';
+import * as path from 'path';
+import { format } from 'util';
+import * as vscode from 'vscode';
+import { OUTPUT_CHANNEL } from '../channels';
+import { nls } from '../messages';
+import { FilePathGatherer, SfdxCommandlet, SfdxWorkspaceChecker } from './util';
+
+const RENAME_LIGHTNING_COMPONENT_EXECUTOR = 'force_rename_lightning_component';
+const RENAME_INPUT_PLACEHOLDER = 'rename_component_input_placeholder';
+const RENAME_INPUT_PROMPT = 'rename_component_input_prompt';
+const RENAME_INPUT_DUP_ERROR = 'rename_component_input_dup_error';
+const RENAME_WARNING = 'rename_component_warning';
+
+export class RenameLwcComponentExecutor extends LibraryCommandletExecutor<string> {
+  private sourceFsPath: string;
+  private responseText: string | undefined;
```

Comment:
Nit: you could make this just `private responseText: string;` and make this class a little simpler. Note you'd also update the constructor parameter.   It doesn't look like we handle when responseText is undefined below so it probably makes sense to to the undefined check before constructing this class instance.

Reply:
@gbockus-sf Revised those part of code to be included in a parameter gatherer.